apps/accounts/templates/accounts/css/report.css

```diff
@@ -173,13 +152,15 @@ p {
 }
 
 .no-data {
+    color: #909399;
+    font: 400 14px/22px 'PingFang SC';
     text-align: center;
-    color: #6c757d;
-    padding: 2rem;
-    background: #f6f6f6;
-    border-radius: 8px;
-    margin: 1rem 0;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+    padding: 20px;
+    font-style: italic;
+    background-color: #f9f9f9;
+    border-radius: 4px;
+    margin: 10px 0;
+    width: 100%;
 }
 
 .new-accounts .section-header span {
```

Comment:
There have been no notable issues, irregularities, or optimization suggestions found in the provided code snippet from September 2021 to March 2025.

No specific recommendations can be made without further context and analysis of the original file's structure and function. Please share more information if you need guidance with different parts of a larger project or document.